Tools/Cli-LoRa-Device-Provisioning/NetIdHelper.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Cli_LoRa_Device_Provisioning
+{
+    using System;
+
+    public static class NetIdHelper
+    {
+        /// <summary>
+        /// Set the NetworkId Part of a devAddr with a given network Id.
+        /// </summary>
+        public static string SetNwkIdPart(string currentDevAddr, string networkId, ConfigurationHelper configurationHelper)
+        {
+            var netIdLastByte = configurationHelper.NetId.Substring(configurationHelper.NetId.Length - 2, 2);
+            uint netId = uint.Parse(netIdLastByte, System.Globalization.NumberStyles.HexNumber);
```

Comment:
change to use TryParse?

Reply:
Nice! I have used my validation function for hex strings instead though to be consistent.